### PR TITLE
use new metadata and show all signals in dashboard

### LINKED ISSUE #926: dashboard: support all API signals
It would be great if the new generic dashboards supports all COVIDcast API signals

### LINKED ISSUE #928: dashboard: support absolute counts for cases/death values
COVIDcast 2.0 is using the per 100k version only. However, for the general dashboard it would be good to have an option to show also the absolute cases/death values in addition to the per100k versions, so that the total variation space includes:

* absolute / per 100k
* cumulative / incidence
* smoothed / raw
* cases / deaths

### LINKED ISSUE #952: Dashboard "geographic region" autocomplete selects wrong region
**Actual Behavior:**

Disregard if this is a problem with a mismatch to the API while we're waiting for nans rollout, but just in case it isn't -- 

When I attempt to add a timeseries widget for Nebraska cases, it gives me "CA - Chico (HRR)" instead. 

**Steps to Reproduce the Bug:**

1. go to https://dev--cmu-delphi-covidcast.netlify.app/dashboard/
2. select "Add Widget"
3. select "Time Series" "COVID Cases"
4. type "Neb" into the Geographic Region field
5. select "Nebraska"
6. click "Add Widget"
7. widget added is for "CA - Chico (HRR)", not "Nebraska"

Oddly enough if you just click in the Geographic Region field and select one of the already-visible options like Alabama, it works fine.

Editing the widget config shows "CA - Chico (HRR)" in the Geographic Region field. If you select Nebraska instead and click Update Widget, the widget remains for "CA - Chico (HRR)". If you type "Chico" and select "CA - Chico (HRR)" and click Update Widget, the widget bizarrely remains for "CA - Chico (HRR)".

"California" works; "Wyoming" does not (yielding "CA - Los Angeles (HRR)" instead).

**Expected behavior**

Resulting widget region should match the selected config.

**Rating Scale (1 is minor, 7 is severe):** Release-blocking, assuming it's not expected to fix itself once we resolve nans

**Context**

- Browser: Chrome Version 91.0.4472.77 (Official Build) (x86_64)

### PR DESCRIPTION
**Pull request by sgratzl**
based on #936 

closes #928, closes #926, closes #952

**Prerequisites**:

- [x] Unless it is a hotfix it should be merged against the `dev` branch
- [x] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [x] Build is successful
- [x] Code is cleaned up and formatted

### Summary

use the new covidcast/meta api to generate a list of sensors that are used in the dashboard. 

Moreover, I changed how the regions/indicator picker is shown

image

image

each geographic level (geo_type) has a color that is also used in the line chart
image

Note: blocked by the epidata release

### CODE REVIEW

**Comment by chinandrew** [comment]
Bug: the y axis labels are all 0 for this JHU signal time series plot
Screenshot_20210520_123447

**Comment by sgratzl** [comment]
> Bug: the y axis labels are all 0 for this JHU signal time series plot

fixed: 
image

**Comment by krivard** [comment]
There's some commented-out code in here; is that intentional?

File: src/modes/modes.js [hunk 1]
```diff
@@ -40,11 +40,11 @@ export const modes = [
     label: 'Export Data',
     component: () => import(/* webpackChunkName: 'm-export' */ './exportdata/ExportData.svelte').then((r) => r.default),
   },
-  {
-    id: 'lab',
-    label: 'Lab',
-    component: () => import(/* webpackChunkName: 'mode-lab' */ './lab/Lab.svelte').then((r) => r.default),
-  },
+  // {
```
Comment (sgratzl) [inline src/modes/modes.js:43]: intentional we might need that again in the future for experiments

**Comment by sgratzl** [comment]
> There's some commented-out code in here; is that intentional?

I just checked again, and removed some old code. Besides that the comments are intentional.

**Comment by krivard** [comment]
Menu item should read "Dashboard Builder"

**Comment by sgratzl** [comment]
> Menu item should read "Dashboard Builder"

see also https://github.com/cmu-delphi/www-main/pull/342

**Review by krivard — APPROVED** [review]
👍

**Pull request merged**